model/host/db.go

```diff
@@ -1254,14 +1255,18 @@ func FindDistroForHost(hostID string) (string, error) {
 	return h.Distro.Id, nil
 }
 
-func FindVolumesByUser(userID string) ([]Volume, error) {
+func FindVolumes(q bson.M) ([]Volume, error) {
```

Comment:
This should probably be a private function

Reply:
done